lib/site_prism/element_container.rb

```diff
@@ -148,8 +148,7 @@ def create_nonexistence_checker(element_name, *find_args)
     def create_waiter(element_name, *find_args)
       method_name = "wait_for_#{element_name}"
       create_helper_method(method_name, *find_args) do
-        define_method(method_name) do |timeout = nil, *runtime_args|
-          timeout = timeout.nil? ? Capybara.default_max_wait_time : timeout
+        define_method(method_name) do |timeout = Capybara.default_max_wait_time, *runtime_args|
```

Comment:
The line is too long. Pls, make it 80.

Reply:
There are some tasks in the todo document to reduce some of the cop infractions. Currently the max line length is set to something like 130.

It needs reducing I agree, but not here.